Ball of Duty Server/Domain/Broker.cs

```diff
@@ -145,11 +243,19 @@ private void Read(ref byte[] buffer)
             }
         }
 
-        public void Send(byte[] b)
+        public void SendTcp(byte[] b)
+        {
+            foreach (AsyncSocket client in _connectedClients.ToArray())
```

Comment:
why copy the elements to an array before iteration? it is a concurrent collection after all